frontend/packages/gitops-plugin/integration-tests/features/gitops/add-manifest-url.feature

```diff
@@ -0,0 +1,48 @@
+@gitOps
+Feature: Applications managed by GitOps and Create secret
+    User should be able to see list of applications managed by GitOps
+
+
+        Background:
+            Given user is at developer perspective
+
+
+        @smoke
+        Scenario: No GitOps Manifest URLs found
+             When user navigates to Environments page
+             Then user will see the message No GitOps manifest URLs found
+
+
+        @regression @manual
+        Scenario: Add Manifest URL and No Application groups found message
+             When user has created namespace "aut-addflow-catalog"
+              And user navigates to Search page
+              And user selects "Namespaces" on Resources dropdown
+              And user selects Namespace "aut-addflow-catalog"
+              And user clicks on YAML tab
+              And user adds annotation "app.openshift.io/vcs-uri"
+              And user adds the Manifest URL under annotation "app.openshift.io/vcs-uri"
+              And user saves the YAML
+              And user reloads the YAML to see the changes
+              And user navigates to Environments page
+             Then user will see No Application groups found message
```

Comment:
This scenario can be removed now as we don't need to add Manifest URLs manually now

Reply:
commented, maybe helpful to have this documentation in future. if someone wants to create it manually